src/Features/CSharp/Portable/CodeRefactorings/ConvertToRecord/CSharpConvertToRecordRefactoringProvider.cs

```diff
@@ -0,0 +1,780 @@
+                        var summaryContent = ((XmlElementSyntax)summaryNode).Content;
+                        paramContent = summaryContent.Select((node, index) =>
+                        {
+                            if (node is XmlTextSyntax text)
+                            {
+                                // any text token that is not on it's own line should have replaced trivia
+                                var tokens = text.TextTokens.SelectAsArray(token =>
+                                    token.IsKind(SyntaxKind.XmlTextLiteralToken)
+                                        ? token.WithLeadingTrivia(exteriorTrivia)
+                                        : token);
+
+                                if (index == 0 &&
+                                    tokens.Length >= 2 &&
+                                    tokens[0].IsKind(SyntaxKind.XmlTextLiteralNewLineToken))
+                                {
+                                    // remove the starting line and trivia from the first line
+                                    tokens = tokens.RemoveAt(0);
+                                    tokens = tokens.Replace(tokens[0], tokens[0].WithoutLeadingTrivia());
+                                }
+
+                                if (index == summaryContent.Count - 1 &&
+                                    tokens.Length >= 2 &&
+                                    tokens[^1].IsKind(SyntaxKind.XmlTextLiteralToken) &&
+                                    tokens[^1].Text.GetFirstNonWhitespaceIndexInString() == -1 &&
+                                    tokens[^2].IsKind(SyntaxKind.XmlTextLiteralNewLineToken))
+                                {
+                                    // the last text token contains a new line, then a whitespace only text (which would start the closing tag)
+                                    // remove the new line and the trivia from the extra text
+                                    tokens = tokens.RemoveAt(tokens.Length - 2);
+                                    tokens = tokens.Replace(tokens[^1], tokens[^1].WithoutLeadingTrivia());
+                                }
+
+                                return text.WithTextTokens(SyntaxFactory.TokenList(tokens));
+                            }
+                            return node;
+                        }).AsImmutable();
+                    }
+                }
+
+                // add an extra line and space with the exterior trivia, so that our params start on the next line and each
+                // param goes on a new line with the continuation trivia
+                // when adding a new line, the continue flag adds a single line documentation trivia, but we don't necessarily want that
+                yield return SyntaxFactory.XmlText(
+                    SyntaxFactory.XmlTextNewLine("\r\n", continueXmlDocumentationComment: false),
```

Comment:
This might be a problem for VS for Mac/VS Code where people might be editing files with different endedness -- we should be trying to get the newline character from somewhere else (either a line in the file, or the formatting option.)